docs/fsharp/language-reference/discriminated-unions.md

```diff
@@ -83,6 +81,29 @@ let getShapeHeight shape =
 
 Normally, the case identifiers can be used without qualifying them with the name of the union. If you want the name to always be qualified with the name of the union, you can apply the [RequireQualifiedAccess](https://msdn.microsoft.com/library/8b9b6ade-0471-4413-ac5d-638cd0de5f15) attribute to the union type definition.
 
+## Struct Discriminated Unions
+
+Starting with F# 4.1, you can also represent Discriminated Unions as structs.  This is done with the `[<Struct>]` attribute.
+
+```fsharp
+[<Struct>]
+type SingleCase = Case of string
+
+[<Struct>]
+type Multicase =
+    | Case1 of string
+    | Case2 of int
+    | Case3 of double
+```
+
+>[!WARN]
+With the initial release of F# 4.1, multicase struct unions have isues if the cases are of the same type.  This has since been fixed and is pending an update.
```

Comment:
typo: isues
wrong note formatting. it should be WARNING not WARN and next line also starts with >